tox.ini

```diff
@@ -5,13 +5,13 @@
 
 [tox]
 envlist =
-    py{27,34,35}-django{18,19},
-    py{27,34,35}-django-master,
+    py{27,34,35}-django{18,19,110,-master},
 
 [testenv]
 commands = {envpython} runtests.py
 deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
+    django110: Django>=1.10a1,<1.11
```

Comment:
Yes, although it could be helpful to leave this pattern so it's obvious how to add support for the next prerelease of Django.